uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-calendar.html

```diff
@@ -0,0 +1,386 @@
+<section>
```

Comment:
Instead of writing the whole HTML statically, can we build this using arrays?
Example:
```
$scope.calendarData = [
 +        {
 +          'date': '2016-01-01',
 +          'type': {
 +            'name': 'working_day',
 +          }
 +        },
 +        {
 +          'date': '2016-01-02',
 +          'type': {
 +            'name': 'non_working_day'
 +          }
 +        },
 +        {
 +          'date': '2016-01-03',
 +          'type': {
 +            'name': 'weekend'
 +          }
 +        }
 +      ]
```

We can use the above data(or something similar) and use `ng-repeat` then the html would be reduced and simplified. Also on the next task it would be easier to integrate with original data.